libultraship/libultraship/SohImGuiImpl.cpp

```diff
@@ -874,6 +874,8 @@ namespace SohImGui {
                     Tooltip("Allows the Lon Lon Ranch obstacle course reward to be shared across time periods");
                     EnhancementCheckbox("Goron Tunic No Fire", "gGoronNoFire");
                     Tooltip("Stops Link from catching fire while wearing the Goron Tunic");
+                    EnhancementCheckbox("Super Bow", "gSuperBow");
+                    Tooltip("Improves your bow!");
```

Comment:
I'm sure @jbodner09 had touched on this previously, but is there a way to elaborate on how this makes your bow "super?"

Reply:
It was previously in a "Fun" Cheat category in the Cheat Menu; it's a joke "cheat", which switches out the normal arrow "swoosh" with King Zora's "Moip!".

Reply:
The tooltip should explain this then.

I'm also not sure this really fits within the cheats menu, or is something that should be a native feature of SoH at all tbh. This is more within the scope of mods.

Reply:
No Problem. I could remove it, and make a mod in that has it instead. ^^